components/Calender/index.vue

```diff
@@ -3,16 +3,16 @@
         <h1>Calender</h1>
         <!-- <CalenderHeader :date="date" @prev="PrevMonth" @next="NextMonth" /> -->
         <!-- <CalenderWeekDay /> -->
-        <div class="calender-body">
-            <div class="calender-item" v-for="item in datesOfMonth" :key="item.Date" :data-date="item.Date"
+        <div class="calender-container">
+            <div class="calender-day" v-for="item in CalenderDays" :key="item.Date" :data-date="item.Date"
                 :style="{ height: `${item.height}px` }" @click="toggleModal(item.Date)">
-                <div class="calender-item-events">
-                    <div class="calender-item-event-date">
-                        <div class="text-center" :class="{ 'text-gray-200': !item.current }">
+                <div class="calender-day-events">
```

Comment:
this div is showing the date in calender so we can rename class to `calenderDate`